TaylorFramework/Modules/finder/Finder.swift

```diff
@@ -25,7 +25,7 @@ final class Finder {
             return []
         }
```

Comment:
Please refactor this statement as follows 
```
guard let parameters = parameters where validateParameters(parameters) else { return [] }
```
Now you can use `parameters` variable everywhere without force-unwrapping it because you are sure it is not `nil`.